scheduler/scheduler-server/src/test/java/org/ow2/proactive/scheduler/signal/SignalApiTest.java

```diff
@@ -310,6 +310,34 @@ public void testWaitForAny() throws SignalApiException {
         Assert.assertFalse(signalApi.isReceived(signal_2));
     }
 
+    @Test
+    public void testWaitForAll() throws SignalApiException {
+        String signal_1 = "test_signal_11_1";
+        String signal_2 = "test_signal_11_2";
+        Set<String> signals = new HashSet<String>() {
+            {
+                add(signal_1);
+                add(signal_2);
+            }
+        };
+        long durationInMillis = 0;
```

Comment:
Are you sure that you want to send signals at the same time with 0 latency ?

Reply:
yes :)